pkg/skaffold/instrumentation/export.go

```diff
@@ -398,15 +405,15 @@ func initIOWriterTracer(w io.Writer) (*sdktrace.TracerProvider, func(context.Con
 
 func initCloudTraceExporterApplicationDefaultCreds() (trace.TracerProvider, func(), error) {
 	otel.SetErrorHandler(errHandler{})
-	return texporter.InstallNewPipeline(
-		[]texporter.Option{
-			texporter.WithProjectID(os.Getenv("GOOGLE_CLOUD_PROJECT")),
-			texporter.WithOnError(func(err error) {
-				log.Entry(context.TODO()).Debugf("Error with metrics: %v", err)
-			}),
-		},
-		sdktrace.WithSampler(sdktrace.AlwaysSample()),
-	)
+	exporter, err := texporter.New()
```

Comment:
this method has already handled default credential fetching.

Reply:
Makes sense